radar-upload-frontend/src/components/Record/RecordTable.vue

```diff
@@ -297,6 +305,13 @@ export default {
       });
       this.recordList.splice(recordIndex, 1);
     },
+    retryRecordUpload({ recordId }) {
+      const recordIndex = this.recordList.findIndex(re => re.id === recordId);
+      fileAPI.retryRecordUpload({
+        recordId
+      });
+      this.getRecordList();
```

Comment:
Would it make sense to fetch the new record list after the retryRecordUpload promise has succeeded?